src/devices/Ccs811/Ccs811Sensor.cs

```diff
@@ -0,0 +1,518 @@
+        public bool WakeEnable => _pinWake >= 0;
+
+        /// <summary>
+        /// Do we have data ready to read?
+        /// </summary>
+        public bool IsDataReady
+        {
+            get
+            {
+                var status = (Status)ReadRegister(Register.STATUS);
+                return status.HasFlag(Status.DATA_READY);
+            }
+        }
+
+        /// <summary>
+        /// Read the equivalent CO2 in ppm and equivalent Total Volatile Compound in ppb
+        /// </summary>
+        /// <param name="equivalentCO2InPpm">The equivalent CO2 (eCO2) output range for CCS811 is from
+        /// 400ppm up to 29206ppm.</param>
+        /// <param name="equivalentTotalVolatilOrganicCompoundInPpb">The equivalent Total Volatile Organic Compound (eTVOC)
+        /// output range for CCS811 is from 0ppb up to 32768ppb</param>
+        /// <param name="rawCurrentSelected">Raw data containing the value of the
+        /// current through the sensor(0μA to 63μA)</param>
+        /// <param name="rawAdcReading">Raw data containing  the
+        /// readings of the voltage across the sensor with the selected
+        /// current(1023 = 1.65V)</param>
+        /// <returns>True if success</returns>
+        public bool ReadGasData(out int equivalentCO2InPpm, out int equivalentTotalVolatilOrganicCompoundInPpb, out int rawCurrentSelected, out int rawAdcReading)
+        {
+            equivalentCO2InPpm = -1;
+            equivalentTotalVolatilOrganicCompoundInPpb = -1;
+            rawCurrentSelected = -1;
+            rawAdcReading = -1;
+            Span<byte> toRead = stackalloc byte[8];
+            ReadRegister(Register.ALG_RESULT_DATA, toRead);
+            if (toRead[5] != (byte)Error.NoError)
+            {
+                return false;
+            }
+
+            equivalentCO2InPpm = BinaryPrimitives.ReadInt16BigEndian(toRead.Slice(0, 2));
+            equivalentTotalVolatilOrganicCompoundInPpb = BinaryPrimitives.ReadInt16BigEndian(toRead.Slice(2, 2));
+            rawCurrentSelected = toRead[6] >> 2;
+            rawAdcReading = ((toRead[6] & 0b0000_0011) << 2) + toRead[7];
+            return ((equivalentCO2InPpm >= 400) && (equivalentCO2InPpm <= 29206) && (equivalentTotalVolatilOrganicCompoundInPpb >= 0) && (equivalentTotalVolatilOrganicCompoundInPpb <= 32768));
+        }
+
+        /// <summary>
+        /// Read the equivalent CO2 in ppm and equivalent Total Volatile Compound in ppb
+        /// </summary>
+        /// <param name="equivalentCO2InPpm">The equivalent CO2 (eCO2) output range for CCS811 is from
+        /// 400ppm up to 29206ppm.</param>
+        /// <param name="equivalentTotalVolatilOrganicCompoundInPpb">The equivalent Total Volatile Organic Compound (eTVOC)
+        /// output range for CCS811 is from 0ppb up to 32768ppb</param>
+        /// <returns>True if success</returns>
+        public bool ReadGasData(out int equivalentCO2InPpm, out int equivalentTotalVolatilOrganicCompoundInPpb)
```

Comment:
I assume UnitsNet doesn't have any typed units for this? `int` is fine if there is no unit since it represents quantity

Reply:
I checked and rechecked now you're asking the question: https://github.com/angularsen/UnitsNet/blob/master/UnitsNet/GeneratedCode/Units/VolumeConcentrationUnit.g.cs
So I'll modify everything to integrate it! Good catch.